src/interval.c

```diff
@@ -374,6 +374,19 @@ ts_get_now_internal(Dimension *open_dim)
 #else
 		Datum now_datum = TimestampTzGetDatum(GetCurrentTimestamp());
 #endif
+
```

Comment:
could you explain why this is a breaking change if the fix is global? We we could defer the fix until we can do this correctly. Any reason not to do that?

Reply:
It's planned to fix this globally for 2.0, but changing the function invalidates already persisted data.